sites/all/modules/custom/dccr/dccr.install

```diff
@@ -60,6 +60,19 @@ function dccr_install() {
   );
 
   $result = module_enable($modules_enable_list, TRUE);
+
+  // Maybe no a nice-looking way; but we want to be able to do a site-install.
+  $actual = 7000;
+  $continue = TRUE;
+  do {
+    $function_name = "dccr_update_$actual";
+    if (function_exists($function_name)) {
+      $function_name();
```

Comment:
please add new line with $actual++
When i add this in my local site the function works, but crash the site:
 PHP Fatal error:  Call to undefined method i18n_object_wrapper::strings_update() in /home/rodrigo/ProyectosPantheon/dccrtest/sites/all/modules/contrib/i18n/i18n_string/i18n_string.module on line 851
Drush command terminated abnormally due to an unrecoverable error.                                                                                                                                                                 [error]
Error: Call to undefined method i18n_object_wrapper::strings_update() in /home/rodrigo/ProyectosPantheon/dccrtest/sites/all/modules/contrib/i18n/i18n_string/i18n_string.module, line 851